### PR TITLE
Add config option to scrub ips from logs

### PR DESCRIPTION
**Pull request by kralverde**
adds an option to scrub IPs from logs

### CODE REVIEW

File: pumpkin/src/main.rs [hunk 1]
```diff
@@ -128,7 +139,10 @@ fn main() -> io::Result<()> {
 
         let server = Arc::new(Server::new());
         log::info!("Started Server took {}ms", time.elapsed().as_millis());
-        log::info!("You now can connect to the server, Listening on {}", addr);
+        log::info!(
+            "You now can connect to the server, Listening on {}",
+            scrub_address(&format!("{}", addr))
+        );
```
Comment (Snowiiii) [inline pumpkin/src/main.rs:145]: I think we should leave this. Everyone can easly see a ip from a domain

**Comment by Snowiiii** [comment]
I also now require config changes to be added to the docs

**Comment by kralverde** [comment]
What files should I had the new config value to?

**Comment by Snowiiii** [comment]
> What files should I had the new config value to?

Editing docs is very simple, They are represented as markdown. The Config can be found at https://github.com/Snowiiii/Pumpkin/tree/master/docs/config

**Comment by kralverde** [comment]
I've added the config and tweaked the headers a-la https://github.com/DavidAnson/markdownlint/blob/main/doc/md001.md

**Comment by Snowiiii** [comment]
> I've added the config and tweaked the headers a-la https://github.com/DavidAnson/markdownlint/blob/main/doc/md001.md

Please do this in an different PR

**Comment by Snowiiii** [comment]
Looks good

**Pull request merged**